deepchecks/vision/dataset.py

```diff
@@ -188,7 +179,36 @@ def get_data_loader(self):
         """Return the data loader."""
         return self._data
 
-    def validate_shared_label(self, other):
+    def get_transform_type(self):
+        """Return transforms handler created from the transform field."""
+        dataset_ref = self.get_data_loader().dataset
+        # If no field exists raise error
+        if not hasattr(dataset_ref, self.transform_field):
+            raise DeepchecksValueError(f'Underlying Dataset instance must have a {self.transform_field} attribute')
```

Comment:
Maybe not it should be the one containing the aug?

Reply:
updated original comment, meant maybe we should say here that this field should contain the aug